packages/cover-traffic-daemon/src/index.ts

```diff
@@ -117,7 +117,7 @@ export async function main(update: (State: State) => void, peerId?: PeerId) {
   node.indexer.on('peer', peerUpdate)
   node.indexer.on('block', (blockNumber) => data.setBlock(new BN(blockNumber.toString())))
 
-  console.log(`Address: ${selfAddr.toHex()}`)
+  log(`Address: ${selfAddr.toHex()}`)
```

Comment:
This could be the source of issue as CT test may be failing to figure the node address

Reply:
Indeed, but still this begs the question on _why_ this works locally.

Reply:
That doesn't seem to be the case, see f0d4fab

Reply:
When changing from `console.log` to `log`, [this line](https://github.com/hoprnet/hoprnet/blob/8c6124c0dbd2657e29485bac8aa65b57aba17fbc/scripts/run-integration-tests-source.sh#L294) also needs to be adapted, from `2` to `4`